{{cookiecutter.repostory_name}}/tools/backup-b2/Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG BASE_IMAGE=python:3.8-alpine
+
+FROM $BASE_IMAGE AS base-image
```

Comment:
this is a good convention in multi-stage builds. It is redundant, but I'd rather keep it. Authors preference wins, I think.

Reply:
unless you strongly disagree?

Reply:
You know me. I don't like redundancy in the code and having something that is not used. It may be confusing for someone new, but I won't strongly disagree with this. It works as expected anyway